util/container/Dockerfile

```diff
@@ -96,6 +99,54 @@ COPY util/get-toolchain.py /tmp/get-toolchain.py
 RUN /tmp/get-toolchain.py -r ${RISCV_TOOLCHAIN_TAR_VERSION} \
     && rm -f /tmp/get-toolchain.py
 
+# Install and configure clang and llvm tools for coverage measurements.
+# Note: Some commands listed below are missing in 13.0.1 and cause warnings during
+# build. These are kept intentionally since we plan to use them in the future, e.g.
+# `llvm-remark-size-diff` and `llvm-tli-checker`.
+RUN curl https://apt.llvm.org/llvm-snapshot.gpg.key | apt-key add \
+    && add-apt-repository "deb http://apt.llvm.org/focal/ llvm-toolchain-focal-${CLANG_VERSION} main" \
+    && apt update \
+    && apt install -y clang-${CLANG_VERSION} lldb-${CLANG_VERSION} lld-${CLANG_VERSION} \
+        clangd-${CLANG_VERSION} clang-tidy-${CLANG_VERSION} clang-format-${CLANG_VERSION} \
+        clang-tools-${CLANG_VERSION} llvm-${CLANG_VERSION} lld-${CLANG_VERSION} \
+        llvm-${CLANG_VERSION}-tools \
+    && ( \
+        primary="llvm-config"; \
+        secondary="llvm-addr2line llvm-ar llvm-as llvm-bcanalyzer llvm-bitcode-strip \
+            llvm-cat llvm-cfi-verify llvm-cov llvm-c-test llvm-cvtres llvm-cxxdump llvm-cxxfilt \
+            llvm-cxxmap llvm-debuginfod llvm-debuginfod-find llvm-diff llvm-dis llvm-dlltool \
+            llvm-dwarfdump llvm-dwarfutil llvm-dwp llvm-exegesis llvm-extract llvm-gsymutil llvm-ifs \
+            llvm-install-name-tool llvm-jitlink llvm-jitlink-executor llvm-lib llvm-libtool-darwin \
+            llvm-link llvm-lipo llvm-lto llvm-lto2 llvm-mc llvm-mca llvm-ml llvm-modextract llvm-mt \
+            llvm-nm llvm-objcopy llvm-objdump llvm-omp-device-info llvm-opt-report llvm-otool \
+            llvm-pdbutil llvm-PerfectShuffle llvm-profdata llvm-profgen llvm-ranlib llvm-rc \
+            llvm-readelf llvm-readobj llvm-reduce llvm-remark-size-diff llvm-rtdyld llvm-sim \
+            llvm-size llvm-split llvm-stress llvm-strings llvm-strip llvm-symbolizer llvm-tapi-diff \
+            llvm-tblgen llvm-tli-checker llvm-undname llvm-windres llvm-xray"; \
+        cmd="--verbose --install /usr/bin/${primary} ${primary} /usr/bin/${primary}-${CLANG_VERSION} 90";\
+        for s in ${secondary}; do \
+            cmd="${cmd} --slave /usr/bin/${s} ${s} /usr/bin/${s}-${CLANG_VERSION}"; \
```

Comment:
Nit: `cmd` threw me off a little because I thought it should be executable, but I saw it began with `--verbose`. WDYT about prefixing with `update-alternatives` and then just executing `${cmd}` on line 130?

Reply:
yeah it doesn't make a lot of sense :) Forgot to cleanup this part while hunting for the list of llvm commands. Done.